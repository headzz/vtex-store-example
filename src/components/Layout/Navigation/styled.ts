import { BLACK } from 'configuration/constants'
import styled from 'styled-components'

export const NavigationStyled = styled.section`
  height: 8rem;
  background-color: ${BLACK};
  width: 100%;
`
