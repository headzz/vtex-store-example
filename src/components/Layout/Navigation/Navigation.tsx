import { NavigationStyled } from './styled'

/**
 * @component
 * @name Navigation
 * @description Defines the container for the layout of navbar  .
 * @namespace components components/Layout/Navigation
 * @param {element} children Component(s) that may appear inside the layout.
 *
 */

const Navigation: React.FC = ({ children }) => {
  return <NavigationStyled>{children}</NavigationStyled>
}

export default Navigation
