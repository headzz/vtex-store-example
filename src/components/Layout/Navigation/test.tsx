import { render } from '@testing-library/react'

import Navigation from './Navigation'

describe('<Navigation />', () => {
  it('should render component that wraps navigation on page', () => {
    const { container } = render(<Navigation />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
