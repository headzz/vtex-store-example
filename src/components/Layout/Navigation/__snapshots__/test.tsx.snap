// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[
  `<Navigation /> should render component that wraps navigation on page 1`
] = `
.c0 {
  height: 8rem;
  background-color: #000000;
  width: 100%;
}

<section
  class="c0"
/>
`
